frontend/app/src/components/buttons/button-primitive.tsx

```diff
@@ -46,7 +46,11 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
     return (
       <button
         type={type}
-        className={classNames(focusStyle, buttonVariants({ variant, size, className }))}
+        className={classNames(
+          "disabled:cursor-not-allowed",
```

Comment:
it should be added inside buttonVariants 1st arg when it's global to all button variants